docs/cleanup.md

```diff
@@ -0,0 +1,35 @@
+# Cleanup
+
+To cleanup TopoLVM, please follow these steps.
+
+**Caution: The cleanup procedure cannot be reverted. Think twice whether you can really cleanup TopoLVM.**
+
+1. Delete all the snapshots and PVCs related to TopoLVM.
+2. If the scheduler extender is enabled, edit the kube-scheduler's config file and remove TopoLVM's settings. It may be your help to read [the configuration guide](https://github.com/topolvm/topolvm/blob/0ccaaa31779c2fd1c51f657a488e39d40afe5b46/deploy/README.md#configure-kube-scheduler).
+   - Please be careful if you have any other settings in extenders section of the configuration file. You can remove only the TopoLVM's setting.
+3. Uninstall TopoLVM via Helm.
+   ```console
+   # Check the namespace
+   helm list -A
+   
+   # Uninstall. Here we assume the namespace is `topolvm-system`.
+   helm uninstall --namespace=topolvm-system topolvm
+   ```
+4. Remove a label from the target namespace.  
+   - When legacy mode is disabled:
+     ```console
+     # Please change the namespace depending on your environment.
+     kubectl label namespace topolvm-system topolvm.io/webhook-
+     kubectl label namespace kube-system topolvm.io/webhook-
+     ```
+   - When legacy mode is enabled:
+     ```console
+     # Please change the namespace depending on your environment.
+     kubectl label namespace topolvm-system topolvm.cybozu.com/webhook-
+     kubectl label namespace kube-system topolvm.cybozu.com/webhook-
+     ```
+5. If LVMd is running as a systemd unit, stop it on each node.
```

Comment:
```suggestion
5. If lvmd is running as a systemd unit, stop it on each node.
```

Just a spelling variants but it seems `lvmd` is official one.
https://github.com/topolvm/topolvm/blob/7da9e8bb7935ff682f14344e491b1f44cda607d2/docs/lvmd.md

Reply:
I changed it to lvmd. Thanks.